### DISCUSSION TITLE
[Request] Configure some attributes ?

### DISCUSSION THREAD

**Discussion by SaiyajinK** [ALTaleX531/OpenGlass#56] [Q&A]
Hi, I have just been informed of the existence of the software, first of all thank you for this one.

Secondly, is it possible to configure certain settings? like the strength of the blur? choose between blur or acrylic ? 🤔

Then I would like to know if it is possible to invert the color of the glass when the explorer window is selected/not selected ?

selected : 
![explorer_RRRTUSuiyO](https://github.com/user-attachments/assets/1db46952-eed2-4872-b5eb-12b0ed1dc1d4)

unselected : 
![explorer_icuwiGnZBW](https://github.com/user-attachments/assets/3ba1750d-dfe4-41ab-8bfb-34cc5dac065c)

I would like that when the window is inactive it is blacker and more blurred, and when it is active lighter with a standard acrylic effect.

I hope it is feasible. Otherwise it does not matter, maybe it will be available in a future version ?

_

**EDIT** : If a simple configuration interface (GUI) is planned for later, I am available to translate it into French.

**Comment by tamarindojuice** [comment]
You can use the AeroGlassGUI.exe from the original glass8 project, just google it and you will find it.
Most options don't work but all the implemented settings on OpenGlass can be changed with easy using it.

![Captura de tela 2024-11-05 134722](https://github.com/user-attachments/assets/1134ca24-05fa-4054-aa37-2100bb751339)

**Comment by SaiyajinK** [comment]
Thanks for the reply. I downloaded from [this](https://archive.org/details/aero-glass-gui.-7z).

Not having a tutorial and having never used this, I put AeroGlassGUI.exe in the OpenGlass folder and open it as admin.

I have this window that appears (I tried different changes on Current user and System-wide settings) :

![image](https://github.com/user-attachments/assets/d1ac9947-e77b-4453-affb-159aeef8969f)

I selected colors on active, inactive things, restarted the pc after saving in the UI but it doesn't seem to work unfortunately.
I tried several parameter combinations, without success.

Unless you have a specific procedure to tell me, I don't understand how to do this. Isn't there any code to modify like in [ExplorerBlurMica](https://github.com/Maplespe/ExplorerBlurMica) ? because the interface doesn't seem to work.

**Comment by ALTaleX531** [comment]
You need to adjust the openglass settings by modifying the corresponding registry items, however you can't configure the different options as active/inactive separately except for the color itself.

There will be no new features nor gui for openglass, if you want it then you will have to modify the code to implement it yourself. As I mentioned in the readme, the person using this software must have a deeper knowledge of windows.

So just wait for the update of dwmblurglass, I heard there will be a major update soon.
